### DISCUSSION TITLE
Translate ScratchAddons to Japanese

### DISCUSSION THREAD

**Discussion by Cyan01-Hash** [ScratchAddons/ScratchAddons#1032] [Translation]
Hello there! I would like to know if I can translate ScratchAddons to **Japanese**.

My **Transifex**: @ScratchRewrite
My **Scratch**: @-Rewrite-

Thanks!

**Comment by apple502j** [comment]
日本語の翻訳はもう済んでいるので大丈夫です。

**Reply by WorldLanguages** [reply]
No idea what it means, but I guess we won't accept Japanese nor Scratch translators because we expect to be active for a while in this project, and having more than one translator might be unproductive?

**Reply by GrahamSH-LLK** [reply]
> It's okay because the Japanese translation has already been completed.
